src/server/seeds/development/06_order_items.js

```diff
@@ -0,0 +1,46 @@
+exports.seed = function (knex) {
+  return knex('order_items')
+    .del()
+    .then(function () {
+      return knex('order_items').insert([
+        {
+          id: 1,
+          order_id: 1,
+          product_id: 1,
+          quantity: 4,
+        },
+        {
+          id: 2,
+          order_id: 1,
+          product_id: 10,
+        },
+        {
+          id: 3,
+          order_id: 2,
+          product_id: 8,
+          quantity: 3,
+        },
+        {
+          id: 4,
+          order_id: 3,
+          product_id: 7,
+        },
+        {
+          id: 5,
+          order_id: 4,
+          product_id: 5,
+        },
+        {
+          id: 6,
+          order_id: 5,
+          product_id: 10,
+        },
```

Comment:
here

Reply:
But i have given the **id** explicitely for **order_items** table, since in the previous **_merged PR_** it was not given as auto increment, but anyway we don't need this **id** for this table. So after merging the PR that have removed the id value, i will also remove this id value for all order items.